src/Lecture1.hs

```diff
@@ -54,7 +56,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Int -> Int -> Int
+sumOfSquares x y = (x * x) + (y * y)
```

Comment:
You don't need `()` here, it will behave the same but it won't hurt to leave them either 🙂